articles/how-to-carry-out-effective-user-acceptance-testing-uat/index.md

```diff
@@ -0,0 +1,60 @@
+
+The diagram below shows where UAT falls in the software development process.
+
+![software process](/engineering-education/how-to-carry-out-effective-user-acceptance-testing-uat/software-process.jpg)
+
+#### Objectives of UAT
+- To confirm that the software product can perform all the business functions.
+- To confirm that the software product is usable from the end-users perspective.
+- To certify that the software product is ready to move to production.
+
+#### Who conducts UAT?
+The end-users or the client conducts UAT. These can be the end-users of a custom made software or someone who has requested custom software from a software developer as a service. The people performing the test are called the test team. In an organization, the team should comprise members from all departments and different groups. The testing team should comprise of subject matter experts. For example, accounting software UAT should involve accountants.
+
+#### UAT Prerequisites
+Some requirements must be met before performing a UAT test. These includes:
+- Provide business requirements to the testing team. UAT is done against business requirements.
+- The system must be fully developed and error-free. Only cosmetic defects are allowed. These are low-priority errors from a business point of view and do not affect the testing process.
+- System testing, integration testing, and unit testing should be completed. System errors are not expected at this stage. The reported flaws are already fixed and evaluated before UAT.
+- UAT environment should be available.
+- [Regression testing](https://www.guru99.com/regression-testing.html) should be complete to confirm that recent code changes do not affect the software.
+
+### Why you need user acceptance testing
+Developers develop software according to their interpretation of the requirements provided. Developers may think they followed the requirements specified only to discover errors after a product is rolled out. It is thus important to confirm that the software is conforming to the business requirements.
+
+There could be some business changes that might not have been communicated to the development team effectively. UAT would be able to capture such.
+
+#### UAT process
+The UAT process can be divided into four phases. In each phase, there are several activities done, as listed below.
+**1. Planning**
+- The list of business processes to be tested is prepared.
```

Comment:
The first process (1. Planning) appears on the first line (see the preview). Click enter to separate it from the first line.